workflows/SUEP_coffea_ZH_simple.py

```diff
@@ -13,7 +13,7 @@
 import numpy as np
 import fastjet
 from coffea import hist, processor, lookup_tools
-import pickle
+import pickle5 as pickle
```

Comment:
No, this breaks our standard coffea installation. Which issues did you have with standard pickle?
Do not change modules like this and push it to our central repository, can very easily break dependencies